### PR TITLE
DOC-960: Update Webhook node to critical tier template

### PR DESCRIPTION
**Pull request by freakwriter**
Summary of changes:

- Convert Webhook node to multi-page layout: move workflow testing/building to separate page, move issues and questions to separate page.
- Add cross-references between pages.
- Update nav.

### CODE REVIEW

File: docs/integrations/builtin/core-nodes/n8n-nodes-base.webhook/common-issues.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+/// note
+In the examples, replace `<https://your-n8n.url/webhook/path>` with your webhook URL.  
+The examples make GET requests. You can use whichever HTTP method you set in **HTTP Method**.
+///
+
+Make an HTTP request without any parameters:
+
+```sh
+curl --request GET <https://your-n8n.url/webhook/path>
```
Comment (imchairmanm) [inline docs/integrations/builtin/core-nodes/n8n-nodes-base.webhook/common-issues.md:46]: Just curious, do we have a way of turning off line numbering on a case-by-case basis?

**Review by imchairmanm — APPROVED** [review]
Had a few questions, but looks good :+1:

File: docs/integrations/builtin/core-nodes/n8n-nodes-base.webhook/common-issues.md (hunk 1 above)
Comment (freakwriter) [reply docs/integrations/builtin/core-nodes/n8n-nodes-base.webhook/common-issues.md:46]: This one I don't know the answer to. You'd need to check how we have mkdocs configured and whether that option or plugin gives you an option to disable line numbering.

File: mkdocs.yml [hunk 2]
```diff
@@ -404,7 +404,10 @@ nav:
         - integrations/builtin/core-nodes/n8n-nodes-base.switch.md
         - integrations/builtin/core-nodes/n8n-nodes-base.totp.md
         - integrations/builtin/core-nodes/n8n-nodes-base.wait.md
-        - integrations/builtin/core-nodes/n8n-nodes-base.webhook.md
+        - Webhook:
+          - integrations/builtin/core-nodes/n8n-nodes-base.webhook/index.md
+          - Common issues: integrations/builtin/core-nodes/n8n-nodes-base.webhook/common-issues.md
+          - Workflow development: integrations/builtin/core-nodes/n8n-nodes-base.webhook/workflow-development.md
```
[the remark this answers is not in the document]
Comment (freakwriter) [reply mkdocs.yml:412]: Oh this was probably a copy/paste fail. Good call, and adjusted.

File: docs/integrations/builtin/core-nodes/n8n-nodes-base.webhook/index.md [hunk 3]
```diff
@@ -11,36 +12,27 @@ hide:
...
-
-<video src="/_video/integrations/builtin/core-nodes/webhook/webhook-node-intro.mp4" controls width="100%"></video>
-
-
-## Production workflows
-
-When your workflow is ready, switch to using the production webhook URL. You can then activate your workflow, and n8n runs it automatically when an external service calls the webhook URL.
-
-When working with a Production webhook, ensure that you have saved and activated the workflow. Data flowing through the webhook isn't visible in the editor UI with the production webhook.
+The Webhook node testing and development process works a bit differently. Refer to [Workflow development](/integrations/builtin/core-nodes/n8n-nodes-base.webhook/workflow-development/) for more information.
```
[the remark this answers is not in the document]
Comment (freakwriter) [reply docs/integrations/builtin/core-nodes/n8n-nodes-base.webhook/index.md:25]: Good point, totally reworded.

**Comment by nik8n** [comment]
One more thing I've seen from users: Not knowing what the test and the production URL is for. Maybe we can add that to the common issues page @freakwriter?

**Pull request merged**